src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -49,41 +47,26 @@ class GroupAiAutofixEndpoint(GroupEndpoint):
         }
     }
 
-    def _get_base_commit(self, group: Group) -> Commit | None:
-        # Using `id__in()` because there is no foreign key relationship.
-        releases_query_set = Release.objects.filter(
-            id__in=GroupRelease.objects.filter(group_id=group.id)
-            .order_by("-last_seen")
-            .values("release_id")
-        )
-
-        if not releases_query_set:
-            return None
+    def _get_repos_from_code_mapping(self, group: Group):
+        repo_configs: list[
+            RepositoryProjectPathConfig
+        ] = RepositoryProjectPathConfig.objects.filter(project__in=[group.project])
 
-        commits: list[Commit] = list(
-            Commit.objects.filter(
-                id__in=ReleaseCommit.objects.filter(release__in=releases_query_set).values("commit")
-            )
-        )
+        repos = []
+        for repo_config in repo_configs:
+            repo: Repository = repo_config.repository
+            repo_name_sections = repo.name.split("/")
 
-        # Hardcoded to only accept getsentry/sentry repo for now, when autofix on the seer side
-        # supports more than just getsentry/sentry, we will just send the latest commit.
-        try:
-            sentry_repo: Repository = Repository.objects.get(
-                organization_id=group.organization.id, name="getsentry/sentry"
-            )
-
-            for commit in commits:
-                if commit.repository_id == sentry_repo.id:
-                    return commit
-        except Repository.DoesNotExist:
-            logger.exception(
-                "No getsentry/sentry repo found for organization",
-                extra={"group.id": group.id, "group.organization.id": group.organization.id},
-            )
-            pass
+            if len(repo_name_sections) == 2:
```

Comment:
what happens if its not 2? should we handle that case?

Reply:
I think github only does 2 sections? But we can make it so it supports as many sections as it comes in but the first section is the owner and the remaining is the name